src/pretix/control/views/item.py

```diff
@@ -1326,7 +1326,8 @@ def plugin_forms(self):
             if isinstance(resp, (list, tuple)):
```

Comment:
Wouldn’t it be easier to just
```suggestion
            if not resp:
                continue
            if isinstance(resp, (list, tuple)):
```